modules/consensus/database.go

```diff
@@ -78,6 +78,29 @@ var (
 	SiafundPool = []byte("SiafundPool")
 )
 
+type (
+	// dbBucket represents a collection of key/value pairs inside the database.
+	dbBucket interface {
+		Get(key []byte) []byte
+	}
+
+	// dbTx represents a read-only transaction on the database that can be used
+	// for retrieving values.
+	dbTx interface {
+		Bucket(name []byte) dbBucket
+	}
+
+	// boltTxWrapper wraps a bolt.Tx so that it matches the dbBucket interface.
```

Comment:
I don't understand the reason for having this when you've got a dbTx interface that could be used.